lib/mixins/vector.js

```diff
@@ -63,14 +63,18 @@ export default {
 
   dash(length, options = {}) {
     let phase;
-    if (length == null) {
-      return this;
-    }
     if (Array.isArray(length)) {
+      const hasZeros = length.filter(x => x === 0).length > 0;
```

Comment:
1. Using either the `every` or `some` function here will be a little more efficient, since they can short-circuit and doesn't build up an intermediate array. 
2. Like the non-array check, this check should reject negative numbers, `NaN`, nulls and other invalid values.

Reply:
👍